server/utils.js

```diff
@@ -577,7 +581,7 @@ function getOrgRestaurants(org, listOfElements) {
 function getUserReviews(user, listOfElements) {
   return objectToArray(listOfElements).filter(
     function(element) {
-      return element.author.name === user;
+      return element.author === user;
```

Comment:
Exactly, this is something I have planned for the next refactorization (all the methods from `orion.js` file)